react-debugging.html

```diff
@@ -65,22 +66,26 @@
         // No need to change anything ABOVE this line ^
 
         return (
-          <h1>Draw a Card</h1>
-          <button onClick={}>
-            Click to Draw a Card
-          </button>
-          <button onclick={shuffle}>
-            Return Cards and Shuffle Deck
-          </button>
-          <div style={handStyles}>
-            {hand.map((card, index) => {
-              return (
+          // missing opening and closing fragment tags         
+          // the send onclick here was missing a capital C to make it onClick
+          // There was an extra { at the end of the return block.
```

Comment:
I recall your talking about this at the assessment review. Glad you notice that you were not following the console. Looks like you did make some corrections that were applicable.